<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="el" sourcelanguage="en">
  <context>
    <name>QObject</name>
    <message>
      <location filename="../scriptingpython.cpp" line="466"/>
      <source>Invalid use of %1 function. Expected %2 arguments, but got %3.</source>
      <translation>Invalid use of %1 function. Expected %2 arguments, but got %3.</translation>
    </message>
    <message>
      <location filename="../scriptingpython.cpp" line="475"/>
      <source>Unknown error from function %1.</source>
      <translation>Unknown error from function %1.</translation>
    </message>
    <message>
      <location filename="../scriptingpython.cpp" line="516"/>
      <source>Could not calculate string representation of the Python object passed as argument to the function %1.</source>
      <translation>Could not calculate string representation of the Python object passed as argument to the function %1.</translation>
    </message>
    <message>
      <location filename="../scriptingpython.cpp" line="532"/>
      <source>Could not find execution context for function %1. This is a bug of Python plugin. Please report it.</source>
      <translation>Could not find execution context for function %1. This is a bug of Python plugin. Please report it.</translation>
    </message>
    <message>
      <location filename="../scriptingpython.cpp" line="557"/>
      <source>Error from Python function %1: %2</source>
      <translation>Error from Python function %1: %2</translation>
    </message>
  </context>
</TS>
